src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2739,19 +2741,23 @@ subroutine CNLivewoodTurnover (num_soilp, filter_soilp, &
             if (CNratio_floating .eqv. .true.) then    
               if (livecrootc(p) == 0.0_r8) then    
                   ntovr = 0.0_r8    
+                  livecrootn_to_deadcrootn(p) = 0.0_r8 
                else    
                   ntovr = ctovr * (livecrootn(p) / livecrootc(p))   
+                   livecrootn_to_deadcrootn(p) = ctovr / deadwdcn(ivt(p)) 
                end if   
 
-               livecrootn_to_deadcrootn(p) = 0.5_r8 * ntovr   ! assuming 50% goes to deadstemn 
+!               livecrootn_to_deadcrootn(p) = 0.5_r8 * ntovr   ! assuming 50% goes to deadstemn 
             end if    
             
             livecrootn_to_retransn(p)  = ntovr - livecrootn_to_deadcrootn(p)
-            if(use_fun)then
-               !TURNED OFF FLUXES TO CORRECT N ACCUMULATION ISSUE. RF. Oct 2015. 
-               livecrootn_to_retransn(p) = 0.0_r8
-               livestemn_to_retransn(p)  = 0.0_r8
-            endif
+
```

Comment:
We should remove commented out code.

So the above changes fix the "N ACCUMULATION ISSUE" Rosie talks about here?

Reply:
These changes do cause an accumulation on N in NPOOL, which isn't ideal, but hopefully less wrong than the previous approach, which results in very low wood C:N ratios.